### ISSUE TITLE
Negative line height inconsistent behavior on Android and iOS 

### ISSUE THREAD

**Issue by jalbatross** [NativeScript/NativeScript#8120]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI:  6.2.0
 - Android Runtime: latest
 - iOS Runtime: latest

**Describe the bug**
On Android, negative line height moves text lines together. On iOS, negative line height has no effect. 

**To Reproduce**
https://play.nativescript.org/?template=play-ng&id=UGy1ZI

**Expected behavior**
The moving text together effect should be the same on both platforms.

**Sample project**
https://play.nativescript.org/?template=play-ng&id=UGy1ZI

**Comment by NickIliev** [comment]
@jalbatross negative values are not officially supported and that's why I am marking the issue as a feature request.

**Comment by alexmeia** [comment]
@NickIliev I agree in making this issue a feature request. Anyway, in my opinion is a very important feature. Negative line-height, although noto official supported, is often used in iOS to give a decent appearance big text in text-wrapped label.

**Comment by alexmeia** [comment]
@jalbatross I need the same feature. Did you find some workaround to set negative line-height in iOS?

**Comment by jalbatross** [comment]
> @jalbatross I need the same feature. Did you find some workaround to set negative line-height in iOS?

I did not find any workaround unfortunately.

**Comment by alexmeia** [comment]
@jalbatross I found one, using native APsI.
I have some titles at font size 66. To reduce the line spacing (really too large at that font size) a call this function when the label is loaded:

```
export function setIOSLineHeight(label: Label, lineHeight: number){
    const iosLabel = label.ios;
    var attributedString;
    if (iosLabel.atributedText) {
        attributedString = iosLabel.atributedText;
    } else {
        attributedString = NSMutableAttributedString.alloc().initWithString(iosLabel.text);
    }
    var range = { location: 0, length: label.text.length };
    const paragraphStyle = NSMutableParagraphStyle.alloc().init();
    paragraphStyle.lineSpacing = 0;
    paragraphStyle.minimumLineHeight = lineHeight;
    paragraphStyle.maximumLineHeight = lineHeight;
    attributedString.addAttributeValueRange(NSParagraphStyleAttributeName, paragraphStyle, range);
    iosLabel.attributedText = attributedString;

}
```
The line-height is set for iOS at the value passed as parameter. For very big font sizes, can be necessary to add some padding-top, because the top part of the first row can be otherwise hidden.

**Comment by newtmex** [comment]
Having the same issue, @alexmeia how do you use your function? How do you call it?

**Comment by alexmeia** [comment]
@newtmex I am using my function and it works. The function is called by the listeners to the `loaded` event of the labels that need this. So, when a label is loaded in a view, the `labelLoaded` method is executed in the controller (`ios:loaded="labelLoaded"`), and inside the `labelLoaded` method is a call to the function described above (I put it in an utility class, so that can be easily shared between app modules).

**Comment by groenroos** [comment]
The above doesn't seem to work at least since NativeScript 7.0. Setting the `attributedText` property simply doesn't seem to have an effect on the Label (confirmed this by trying to set the `attributedText` into some arbitrary text, and it nevertheless uses the plain `text` property).

Is there any new solution that would work with NS 7/8?
